### PR TITLE
Make containers e2e test not complain on unexpected tags on nginx

### PR DESCRIPTION
**Pull request by L3n41c**
### What does this PR do?

Make the `containers` new-e2e tests ignore unexpected tags on `nginx` and `redis` metrics.

### Motivation

If @DataDog/agent-integrations team modify the `nginx` or `redis` checks and make them send legit new tags, we don’t want to impact the `containers` e2e tests.

### Possible Drawbacks / Trade-offs

This effectively reduces the test coverage but `nginx` and `redis` tags should be asserted elsewhere.

### Describe how to test/QA your changes

Check the `new-e2e-containers` GitLab job in the CI pipeline.

### CODE REVIEW

**Review by clamoriniere — APPROVED** [review]

File: test/new-e2e/tests/containers/ecs_test.go [hunk 1]
```diff
@@ -257,15 +258,13 @@ func (suite *ecsSuite) TestRedisECS() {
...
-				`^redis_host:`,
-				`^redis_port:6379$`,
-				`^redis_role:master$`,
 				`^short_image:redis$`,
 				`^task_arn:`,
 				`^task_family:.*-redis-ec2$`,
 				`^task_name:.*-redis-ec2$`,
 				`^task_version:[[:digit:]]+$`,
 			},
+			AcceptUnexpectedTags: true,
```
Comment (pducolin) [inline test/new-e2e/tests/containers/ecs_test.go:267]: **💭 thought**
Wonder if you should allow for a certain pattern of tags, here for instance keys with `redis*`. We should catch reserved tags such as `host` https://docs.datadoghq.com/getting_started/tagging/

Comment (L3n41c) [reply test/new-e2e/tests/containers/ecs_test.go:267]: If we had a clear way to name tags to know what is their source, I would have agreed with you.
The problem is that there isn’t.
For ex., in the context of **incident-25306**, DataDog/integrations-core#16797 added a `port` tag to `nginx` metrics. This tag doesn’t follow any pattern and isn’t prefixed by `nginx_`.
So, in this case, I can’t see how I could have restricted what are “acceptable unexpected tags” to make `port` accepted but not something else.

Comment (iliakur) [reply test/new-e2e/tests/containers/ecs_test.go:267]: Indeed, only metrics have namespaces. For customers this narrows down the tags enough that I guess nobody felt the need to ask for namespaces on tags too.

**Review by iliakur — COMMENTED** [review]
@L3n41c  friendly bump on this, as I'm gonna add back the nginx tags soon ;)

**Pull request merged**